Package.swift

```diff
@@ -0,0 +1,105 @@
+// swift-tools-version:5.1
+import PackageDescription
+
+let package = Package(
+    name: "XMPPFramework",
+    platforms: [
+        .macOS(.v10_10),
+        .iOS(.v8),
+        .tvOS(.v9)
+    ],
+    products: [
+        .library(
+            name: "XMPPFramework",
+            targets: ["XMPPFramework"]
+        ),
+        .library(
+            name: "XMPPFrameworkSwift",
+            targets: ["XMPPFrameworkSwift"]
+        )
+    ],
+    dependencies: [
+        .package(url: "https://github.com/CocoaLumberjack/CocoaLumberjack.git", .upToNextMajor(from: "3.6.1")),
+        .package(url: "https://github.com/robbiehanson/CocoaAsyncSocket.git", .upToNextMajor(from: "7.6.4")),
+        .package(url: "https://github.com/karimhm/KissXML.git", .branch("swift-pm")),
+        .package(url: "https://github.com/karimhm/libidn-framework.git", .branch("swift_pm"))
```

Comment:
Those need to be replaced once **KissXML: [#119](https://github.com/robbiehanson/KissXML/pull/119)** and **libidn-framework: [#6](https://github.com/chrisballinger/libidn-framework/pull/6)** get merged.